src/main/proto/netflix/titus/titus_job_api.proto

```diff
@@ -71,6 +71,22 @@ message Image {
     string digest = 3;
 }
 
+message NetworkConfiguration {
+    enum NetworkMode {
+        // Provides no network access to a container (Not supported yet)
+        None = 0;
```

Comment:
Shouldn't the sentinel value be the default because proto3 always shows the sentinel value?

also, alternatively:

bool ipv4 = 1;
bool ipv6 = 2;
bool transitionMechanism = 3; // If this is specified, and IPv4 and IPv6 are not specified, all traffic will be routed via a transition mechanism system.

Reply:
Hmm right. 
Good thing we are setting this early before too much drift. I'll make ipv4 the default.

Reply:
Er actually, UNKNOWN is a safe default here, and on the backend we can look at the attributes and pick whatever mode they would have gotten in the default case, until spinnaker starts using it.